src/main/java/com/teragrep/pth10/steps/teragrep/bloomfilter/BloomFilterTable.java

```diff
@@ -60,15 +61,26 @@ public final class BloomFilterTable {
     private final LazyConnection conn;
 
     public BloomFilterTable(Config config) {
-        this(new FilterTypes(config), new LazyConnection(config), false);
+        this(config, "default_table_name", new LazyConnection(config), false);
```

Comment:
should the table name always be required, or in which case this "default_table_name" would be used?

Reply:
removed, only constructors with set table values allowed.